rafs/src/metadata/mod.rs

```diff
@@ -397,34 +399,44 @@ pub struct RafsSuperConfig {
     pub is_tarfs_mode: bool,
 }
 
+#[derive(Error, Debug)]
+pub enum MergeError {
```

Comment:
How about just using this:

```
pub enum MergeError {
    #[error("Inconsistent RAFS Filesystem: {0}")]
    InconsisentFilesystem(String),
    #[error(transparent)]
    Other(#[from] Error),
}
```

Reply:
ok, I droped the `DifferentDigestAlgorithm(String)`